RadixWallet/Features/AppFeature/Overlay/InfoLinkSheet+Reducer.swift

```diff
@@ -159,6 +161,10 @@ extension InfoLinkSheet.GlossaryItem {
 			L10n.InfoLink.Glossary.preauthorizations
 		case .possibledappcalls:
 			L10n.InfoLink.Glossary.possibledappcalls
+		case .aboutsecurityshields:
```

Comment:
I think this will be named `securityshield`, according to line 41 of this [doc](https://docs.google.com/spreadsheets/d/1TLMhMdiLLACklRgoqrAUNcr2Yra8uRRratJV4xfrPuw/edit?gid=0#gid=0)